enterprise/internal/database/perms_store.go

```diff
@@ -871,9 +871,21 @@ DO UPDATE SET
    unrestricted = %s;
 `
 
-	q := sqlf.Sprintf(format, pq.Array(ids), unrestricted, unrestricted)
+	size := 65535 / 2 // 65535 is the max number of parameters in a query, 2 is the number of parameters in each row
```

Comment:
```suggestion
	size := 65534 / 2 // 65535 is the max number of parameters in a query, 2 is the number of parameters in each row. We round it down to get an integer.
```
I've checked the `SplitIntoChunks` code to see how it works with values like `32767.5`, but based on the code, I'm not sure, plus I think we could save a lot of wondering by just saying `65534/2` instead.

Reply:
It's go, so integer division leads to what would in javascript be `math.floor(65535/2)`.